pkg/helper/helpers.go

```diff
@@ -124,3 +124,18 @@ func FindFilter(labels []string, isNumber bool) bool {
 
 	return true
 }
+
+func LabelIsHighCardinality(label string) bool {
+	pluginCfg := config.PluginConfig{}
+	err := yaml.Unmarshal(config.LoadStaticFrontendConfig(), &pluginCfg.Frontend)
```

Comment:
It would be good to cache the PluginConfig rather than unmarshaling it every time (not just here but also done in other places). Especially here that's done in a `for` loop for each label

Reply:
we could leverage https://github.com/netobserv/network-observability-operator/blob/main/pkg/helper/helpers.go#L95 remove the for loop and pass in another pool `isHighCardinality` this will do unmarshall once ?

Reply:
I modified `LoadStaticFrontendConfig` to unmarshall inside the function and only once.